koku/api/query_handler.py

```diff
@@ -320,12 +320,11 @@ def filter_to_order_by(self, parameters):  # noqa: C901
             parameters (QueryParameters): The parameters object
 
         """
-        for group_by_key in parameters.parameters.get("group_by", {}):
-            for group_by_value in parameters.parameters["group_by"][group_by_key]:
-                if group_by_value == "*":
-                    # find if there is a filter[X]=Y that matches this group_by[X]=*
-                    # get filter value for current group_by_key
-                    filter_value = parameters.parameters.get("filter", {}).get(group_by_key)
-                    if filter_value:
-                        parameters.parameters["group_by"][group_by_key] = filter_value
+        for key, value in parameters.parameters.get("group_by", {}).items():
```

Comment:
This method is fun with two dictionaries.  What could we do to clarify the meaning of key/value?  I think either we need an update to that comment or rename key/value to have a more specific meaning.

Reply:
Look at the method's docstring. I think that contains the info you're looking for.

Reply:
Looks good